joomla/joomla-cms
# [4.0] Icon- TO fa ( part 3 )

Pull Request for Issue # .

### Summary of Changes
This is the final part of the icon to fa- conversions
converts the remaining instances of icon- to fa- so that we're using fontawesome as core icon class.
*mappings are not changed*


### Testing Instructions
using dev inspector check that new and cancel  in toolbar use "fas fa-" and do display icons.
check all modals the same way.
**ALL icons should now properly use fas fa-**

### Expected result
![image](https://user-images.githubusercontent.com/1850089/75246694-ef044700-5795-11ea-8f48-9f84ad7de9cb.png)


### Actual result
![image](https://user-images.githubusercontent.com/1850089/75246572-a9e01500-5795-11ea-8661-c6fe1e4c319c.png)


### Documentation Changes Required
explain how fontawesome is now the defacto iconset for J4+
to mark a icon with the featured color requires adding "featured" to the icon class.
for example, 
`<i class="fas fa-star"></i>` will result in the uncolorized version.
`<i class="fas fa-star featured"></i> ` will colorize it with the standard icon-featured color
if the full class name is sent instead of just the icon name like 'star' it will be passed thru.  So you could use 'fas fa-cog' or 'icon-cog' and thats what will be used.
